src/routes/(infoPages)/[topTierPage]/+page.server.ts

```diff
@@ -90,6 +95,30 @@ const query = gql`
             slug
           }
         }
+        recentNewsCollection(where: { publicationDate_gt: $newsOldestDate }) {
+          items {
+            sys {
+              id
+            }
+            type
+            title
+            subhead
+            publicationDate
+            slug
+            byline
+          }
+        }
+        upcomingEventsCollection(where: { eventDateAndTime_gte: $eventStartDate }) {
+          items {
+            sys {
+              id
+            }
+            slug
+            shortTitle
+            eventDescription
+            eventDateAndTime
+          }
+        }
```

Comment:
We should probably also set a `limit` here of some sort. I think 5 each of news and events should be sufficient.